cvat-ui/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "src/index.tsx",
   "scripts": {
     "build": "webpack --config ./webpack.config.js",
-    "start": "REACT_APP_API_URL=http://localhost:7000 webpack-dev-server --config ./webpack.config.js --mode=development",
+    "start": "REACT_APP_API_URL=http://localhost:7000 INFURA_ID=a6afd04d29e242be93a6ce29abb4a1ea webpack-dev-server --config ./webpack.config.js --mode=development",
```

Comment:
please, remove this env variable from the package.json
you're passing it via docker compose